third_party/terraform/tests/data_source_google_folder_organization_policy_test.go

```diff
@@ -0,0 +1,93 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceGoogleFolderOrganizationPolicy_basic(t *testing.T) {
+	folder := acctest.RandomWithPrefix("tf-test")
+	org := getTestOrgFromEnv(t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceGoogleFolderOrganizationPolicy_basic(org, folder),
+				Check: testAccDataSourceGoogleOrganizationPolicyCheck(
+					"data.google_folder_organization_policy.data",
+					"google_folder_organization_policy.resource"),
+			},
+		},
+	})
+}
+
+func testAccDataSourceGoogleOrganizationPolicyCheck(dataSourceName string, resourceName string) resource.TestCheckFunc {
```

Comment:
I'm partial to namespacing this & duplicating it into each datasource, but don't care enough to block on it

Reply:
I was thinking the right move would be to make a fully generic comparison, or generate them. But for now I think it's sixes.